Checks/PWR003/solution.c

```diff
@@ -0,0 +1,14 @@
+// PWR003: Explicitly declare pure functions
+
+// Depends only on its arguments
+// No side effects
+
+__attribute__((const)) double gravity();
+
+// Calculate the Weight from mass using gravity (On Earth)
+// W = m · g;
```

Comment:
```suggestion
// Computes the weight of each object in a vector
```

I would get rid of the Unicode characters.